### PR TITLE
gpio inputs in DT

### PR DESCRIPTION
**Pull request by tpaschidis**
Convert gpio direct acces to leds interface

### CODE REVIEW

**Review by dnietotwonav — CHANGES_REQUESTED** [review]
Falta subir a 1.1.33?
**twonav_kernel_version**

**Review by dnietotwonav — APPROVED** [review]

File: arch/arm/boot/dts/stonehenge-big-common.dtsi [hunk 1]
```diff
@@ -73,6 +73,21 @@
 			gpios = <&gpio1 14 GPIO_ACTIVE_HIGH>;
 			default-state = "off";
 		};
+		sd_switch {
+			label = "sd_switch";
+			gpios = <&gpio4 23 GPIO_ACTIVE_HIGH>;
+			default-state = "off";
+		};
+		gprs_switch {
```
Comment (lluisdiaz) [inline arch/arm/boot/dts/stonehenge-big-common.dtsi:81]: Ad far as I know, there's no reason for putting gprs gpio's under common stonehenge. 
About sd_switch, is ok if we want to sell "aventura" as "trail", but putting gpio's as outputs 
could increase power consumption unnecessarily

File: arch/arm/boot/dts/stonehenge-big-common.dtsi [hunk 2]
```diff
@@ -133,23 +164,26 @@
 				/*-------------------------------GPRS----------------------------------*/
 				MX6UL_PAD_ENET2_TX_CLK__GPIO2_IO14		0x1b0b0 /* GPRS_SWITCH */
 				MX6UL_PAD_ENET2_TX_EN__GPIO2_IO13		0x1b0b0 /* GPRS_PWRKEY */
-				MX6UL_PAD_ENET2_TX_DATA0__GPIO2_IO11	0x1b0b0 /* GPRS_STATUS (INPUT) */								
-				/*-------------------------------audio switch--------------------------------*/
+				MX6UL_PAD_ENET2_TX_DATA0__GPIO2_IO11    0x1b0b0 /* GPRS_STATUS (INPUT) */
```
Comment (lluisdiaz) [inline arch/arm/boot/dts/stonehenge-big-common.dtsi:167]: same about gprs gpio's

**Review by lluisdiaz — CHANGES_REQUESTED** [review]
LGTM

**Review by lluisdiaz — APPROVED** [review]

**Review by faragon2n — COMMENTED** [review]
LGTM

**Pull request merged**